### PR TITLE
Fix typo under torch/csrc/jit/runtime directory

### PR DESCRIPTION
**Pull request by kiszk**
This PR fixes typo in comments and messages under `torch/csrc/jit/runtime` directory.

### CODE REVIEW

File: torch/csrc/jit/runtime/serialized_shape_function_registry.cpp [hunk 1]
```diff
@@ -898,8 +898,8 @@ def mv(self: List[int],
   _1 = "AssertionError: max_pool2d: stride must either be omitted, a single int, or a tuple of two ints"
```
Comment (davidberard98) [inline torch/csrc/jit/runtime/serialized_shape_function_registry.cpp:898]: can we leave out this file? it's codegen-ed so these changes will be removed the next time someone regenerates the file.

Comment (kiszk) [reply torch/csrc/jit/runtime/serialized_shape_function_registry.cpp]: Sure, just reverted now.

**Review by davidberard98 — APPROVED** [review]

**Pull request closed without merging**